pandas/io/sql.py

```diff
@@ -140,10 +140,14 @@ def execute(sql, con, cur=None, params=None):
     ----------
     sql : string
         SQL query to be executed.
-    con : SQLAlchemy connectable(engine/connection) or sqlite3 connection
-        Using SQLAlchemy makes it possible to use any DB supported by the
-        library.
-        If a DBAPI2 object, only sqlite3 is supported.
+    con : SQLAlchemy connectable (engine/connection) or database string URI
+        or DBAPI2 connection (fallback mode)
+        Using SQLAlchemy makes it possible to use any DB supported by that
+        library. Legacy support is provided for sqlite3.Connection objects.
+        Note: The user is responsible for engine disposal and connection
```

Comment:
```suggestion

        Note: The user is responsible for engine disposal and connection
```

Reply:
For these "Note:" sections, let's add a newline above (this applies to your other changes)

Reply:
Hey I added the newline however its not rendering in the documentation generated.

Reply:
I think they end up in the same paragraph in the generated HTML.